charmtools/build/tactics.py

```diff
@@ -989,7 +989,7 @@ def _try_to_get_current_sha(self):
                     sha = utils.Process(cmd)()
                     if sha:
                         return sha.output
-                except FileNotFoundError as e:
+                except IOError as e:
                     log.debug(e)
```

Comment:
What you have will ignore diskfull and other exceptions we don't want to catch.
```suggestion
                    if e.errno != errno.ENOENT:
                        raise
                    log.debug(e)
```

Reply:
+1